package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@zimbra/api-client",
 	"amdName": "zmApiJsClient",
-	"version": "36.1.0",
+	"version": "36.2.0",
```

Comment:
why do we need this?

Reply:
I am not sure.I just did rebase and It came from there. In blame, showing  Yash wrote this.
In masters also its showing the same version